<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_8">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>TEXTO GCODE:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonZmenos">
    <property name="geometry">
     <rect>
      <x>222</x>
      <y>131</y>
      <width>45</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Z-</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_9">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>0</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Californian FB&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>INTERFAZ MAQUINA CNC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Cargar Gcode</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>BAUDIOS:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>PUERTO:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>292</x>
      <y>0</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>VISUALIZACION DEL GRAFICO A DIBUJAR:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Importar SVG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonXmas">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>160</y>
      <width>46</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>X+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonZmas">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>131</y>
      <width>46</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);
</string>
    </property>
    <property name="text">
     <string>Z+</string>
    </property>
   </widget>
   <widget class="QComboBox" name="baudOptions">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);</string>
    </property>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="botonYmas">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>131</y>
      <width>46</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Y+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonXmenos">
    <property name="geometry">
     <rect>
      <x>222</x>
      <y>160</y>
      <width>45</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>X-</string>
    </property>
   </widget>
   <widget class="QComboBox" name="portOptions">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);</string>
    </property>
    <item>
     <property name="text">
      <string>COM8</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="botonYmenos">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>46</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Y-</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>291</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLineEdit" name="inputEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>291</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="viewLayout"/>
   </widget>
   <widget class="QPushButton" name="dibujar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>PLAY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="camara">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Captura</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
